src/JuliusSweetland.OptiKey.Core/Services/Translation/TranslationService.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+using System.Net.Http;
+using System.IO;
+using Newtonsoft.Json;
+using System.CodeDom.Compiler;
+using System.CodeDom;
+using JuliusSweetland.OptiKey.Properties;
+
+namespace JuliusSweetland.OptiKey.Services.Translation
+{
+    public class TranslationService
+    {
+        private HttpClient client;
+        private string apiKeyToUse;
+        public const string LICENSE_TEXT = "(Powered by Yandex.Translate)";
+
+        public TranslationService(HttpClient httpClient)
+        {
+            this.client = httpClient;
+        }
+
+        public struct Response
+        {
+            public string status;
+            public string translatedText;
+            public string exceptionMessage;
+        }
+        
+        /*
+         * Returns the translation status, translated text and exception content
+         */
+        public async Task<Response> translate(string text)
+        {
+            Response response;
+            response.status = ""; response.translatedText = ""; response.exceptionMessage = "";
```

Comment:
Does this work? From just looking at the code the response variable would be uninitialised so setting properties on it should cause a NullReferenceException, no?

Either way could this line be split over 3 lines.